apps/prairielearn/src/tests/accessAsStudent.test.ts

```diff
@@ -0,0 +1,61 @@
+import * as cheerio from 'cheerio';
+import { assert } from 'chai';
```

Comment:
@nwalters512 I see this reference to chai assert instead of the assert library. Any preference? Should I replace?

Reply:
We should basically always be using `chai` instead of `node:assert`. Ironically, this file is correct, and the other two tests you modifed are the only two test files that use `node:assert` directly 😛